modular_skyrat/modules/Midroundtraitor/code/datum_traitor.dm

```diff
@@ -3,6 +3,7 @@
 	var/infil_outfit = /datum/outfit/syndicateinfiltrator
 	preview_outfit = /datum/outfit/lone_infiltrator_preview
 	job_rank = ROLE_LONE_INFILTRATOR
+	progression_enabled = TRUE
```

Comment:
And why did the infiltrator have prog-tot objectives, again? Wasn't the whole point literally not to have prog-tots anymore?

Reply:
*(discussed and resolved on-discord, an oversight from the original progtot PR)*